physionet-django/console/views.py

```diff
@@ -1157,6 +1158,15 @@ def process_credential_application(request, application_slug):
                                                send=False,  wordwrap=False)
     contact_cred_ref_form = forms.ContactCredentialRefForm(initial=ref_email)
 
+    try:
+        ref = User.objects.get(associated_emails__email=application.reference_email.lower(),
```

Comment:
Sorry, my bad, there is a case insensitive search term (`email__iexact="applications.reference_...`) which seems better than converting to lower()?